content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Announcing Kubernetes Upstream Community Group Annual Reports"
+date: 2021-06-18T10:00:00-08:00
+slug: Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports
+---
+
+**Authors:** Divya Mohan
+
+Given the growth and scale of the Kubernetes project, the existing reporting mechanisms were proving to be inadequate & challenging. The sheer number of community groups (37 at the time of writing this), their varied sources & types of data that needed to be incorporated while preparing such updates, & the delivery of those (how many meetings are too many meetings?) clearly were a call to action for exploring better options. There are over 300 GitHub repositories alone and we count more than just commits to these repositories as a contribution. Therefore inspired by the Apache Software Foundation’s open guide to PMC Reporting and the CNCF project Annual Reporting, the Kubernetes project is proud to announce the **Kubernetes Community Group Annual Reports for Special Interest Groups (SIGs) and Working Groups (WGs)**. In its flagship edition, the [2020 Summary report]() focuses on bettering the Kubernetes ecosystem by assessing and promoting the healthiness of the groups within the upstream community. 
```

Comment:
>  existing reporting mechanisms

Readers might wonder what these were / when those will be explained (I did).

Reply:
Would it help to hyperlink to https://www.apache.org/foundation/board/reporting and to the equivalent CNCF document?

Reply:
> The sheer number of community groups (37 at the time of writing this), their varied sources & types of data that needed to be incorporated while preparing such updates, & the delivery of those (how many meetings are too many meetings?) clearly were a call to action for exploring better options.

How about:
> Kubernetes is a large open source project. With over 100000 commits just to the main [k/kubernetes](https://github.com/kubernetes/kubernetes) repository, hundreds of other code repositories in the project, and thousands of contributors, there's a lot going on. In fact,
there are 37 contributor groups at the time of writing. We also value all forms of contribution and not just code changes.
> 
> With that context in mind, the challenge of reporting on all this activity was a call to action for exploring better options.

?

I picked “contributor groups” to avoid this being confused with end-user meetups etc.

Reply:
@parispittman : Please could you chip in?

Reply:
+1 to contributor groups and Tim's suggestion